in_memory_person_storage.go

```diff
@@ -21,9 +21,58 @@ func (s *InMemoryPersonStorage) Add(person *Person) error {
 }
 
 func (s *InMemoryPersonStorage) GetAll() []*Person {
-	persons := make([]*Person, 0)
+	var persons []*Person
 	for _, person := range s.data {
 		persons = append(persons, person)
 	}
 	return persons
 }
+
+func (s *InMemoryPersonStorage) GetPersonById(id uuid.UUID) *Person {
```

Comment:
it's a common pattern to return error from mostly all methods.
if person was not found, it is also common to return not found error error